src/Microsoft.Identity.Web/Extensions.cs

```diff
@@ -17,7 +19,7 @@ public static bool ContainsAny(this string searchFor, params string[] stringColl
         {
             foreach (string str in stringCollection)
             {
-                if (searchFor.Contains(str))
+                if (searchFor.Contains(str, StringComparison.OrdinalIgnoreCase))
```

Comment:
do we really want it to be case insensitive in all cases?

Reply:
i'm not sure.

Reply:
To avoid bugs like #1163 
The strings we use in our libs are mostly URIs, settings or env variables. We treat them all in a case-insensitive way.

Reply:
https://www.joelonsoftware.com/2003/10/08/the-absolute-minimum-every-software-developer-absolutely-positively-must-know-about-unicode-and-character-sets-no-excuses/